gitlab/assets/dashboards/overview.json

```diff
@@ -0,0 +1,419 @@
+        {
+            "height": 9,
+            "id": 0,
+            "sizing": "zoom",
+            "type": "image",
+            "url": "https://static.datadoghq.com/static/images/saas_logos/bot/gitlab@2x.png",
+            "width": 15,
+            "x": 1,
+            "y": 1
+        },
+        {
+            "check": "gitlab.liveness",
+            "group": "gitlab_host:localhost,gitlab_port:8086,optional:tag1,host:docker-desktop",
+            "group_by": [],
+            "grouping": "cluster",
+            "height": 9,
+            "id": 1,
+            "tags": [
+                "*"
+            ],
+            "title": true,
+            "title_align": "center",
+            "title_size": "16",
+            "title_text": "Liveness",
+            "type": "check_status",
+            "width": 14,
+            "x": 18,
+            "y": 1
+        },
+        {
+            "check": "gitlab.readiness",
+            "group_by": [],
+            "grouping": "cluster",
+            "height": 8,
+            "id": 2,
+            "tags": [
+                "*"
+            ],
+            "title": true,
+            "title_align": "center",
+            "title_size": "16",
+            "title_text": "Readiness",
+            "type": "check_status",
+            "width": 14,
+            "x": 18,
+            "y": 11
+        },
+        {
+            "check": "gitlab.health",
+            "group_by": [],
+            "grouping": "cluster",
+            "height": 8,
+            "id": 3,
+            "tags": [
+                "*"
+            ],
+            "title": true,
+            "title_align": "center",
+            "title_size": "16",
+            "title_text": "Health",
+            "type": "check_status",
+            "width": 14,
+            "x": 18,
+            "y": 20
+        },
+        {
+            "check": "gitlab.prometheus_endpoint_up",
+            "group_by": [],
+            "grouping": "cluster",
+            "height": 8,
+            "id": 4,
+            "tags": [
+                "*"
+            ],
+            "title": true,
+            "title_align": "center",
+            "title_size": "16",
+            "title_text": "Endpoint Up",
+            "type": "check_status",
+            "width": 14,
+            "x": 18,
+            "y": 29
+        },
+        {
+            "bgcolor": "gray",
+            "font_size": "18",
+            "height": 25,
+            "html": "Service Checks",
+            "id": 5,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "right",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 15,
+            "x": 1,
+            "y": 12
+        },
+        {
+            "height": 13,
+            "id": 6,
+            "legend": false,
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:gitlab.rack.http_requests_total{*} by {method}.as_count()",
```

Comment:
- Ditto template variables.
- Thinking that should probably be a sum instead of avg here.
The `avg:` part defines the space-aggregation algorithm to use i.e when there are multiple timeseries with different tags, how are those aggregated.
IMO, if we have multipe gitlab host exposing that metrics, we should display the total number of http requests across hosts.